src/constant.ts

```diff
@@ -0,0 +1,97 @@
+export const bsBaseYear = 2000; // The base year for BS (2000 BS corresponds to 1943 AD)
+export const adBaseYear = 1943; // The base year for AD
+
+// 2D array where each inner array corresponds to a BS year
+export const bsMonths: number[][] = [
```

Comment:
can you compare this with https://github.com/techgaun/ad-bs-converter/blob/master/src/converter.js#L128 and add the data from the converter I had built?